common/capabilities/channel.go

```diff
@@ -61,6 +66,8 @@ func (cp *ChannelProvider) Type() string {
 func (cp *ChannelProvider) HasCapability(capability string) bool {
 	switch capability {
 	// Add new capability names here
+	case ChannelV3_0:
+		return false // Will be enabled in future v3.x binaries
```

Comment:
1. `main` is now targeting v3.0 so this can be `true`.
2. What about `configtx.yaml` changes? You had them in your PR which you cherry picked from